prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -1476,12 +1476,24 @@ class Hl7Serializer(
             "FTS|1$hl7SegmentDelimiter"
     }
 
+    /**
+     * Given an hl7Configuration, this will take find the current date time and output it to a
+     * specific format depending on the configuration of the receiver
+     */
     private fun nowTimestamp(hl7Config: Hl7Configuration? = null): String {
+        // get the current time stamp
```

Comment:
Add unit tests

Reply:
Added. Please rereview.

Reply:
Thank you, it looks good.  I reviewed and tested.